localstack/services/sqs/provider.py

```diff
@@ -215,10 +227,9 @@ class SqsQueue:
     inflight: Set[SqsMessage]
     receipts: Dict[str, SqsMessage]
 
-    def __init__(self, key: QueueKey, attributes=None, tags=None) -> None:
-        super().__init__()
-        self._assert_queue_name(key.name)
-        self.key = key
+    def __init__(self, name: str, attributes=None, tags=None) -> None:
+        self.name = name
+        self._assert_queue_name(name)
```

Comment:
i would prefer to keep the information which region and account the queue belongs to in the resource. #5928 introduces a significant change here that quires the account id and the region to be stored in the queue. if we want to remove the `QueueKey` as a concept, then pass the region and account together with the name.

Reply:
Good idea, made `account_id` and `region` attributes